src/components/Button/index.tsx

```diff
@@ -1,19 +1,30 @@
-import { h } from 'preact'
+import { h, FunctionComponent } from 'preact'
+import { HTMLProps } from 'react'
 import classNames from 'classnames'
-import { isDesktop } from '~utils/index'
+import { isDesktop } from '~utils'
 import style from './style.scss'
 
-const Button = ({
-  uiTestDataAttribute,
+type Props = HTMLProps<HTMLButtonElement> & {
+  ariaBusy?: string
+  ariaLive?: string
+  ariaRelevant?: string
+  onClick?: h.JSX.MouseEventHandler<HTMLButtonElement>
+  textClassName?: string
+  uiTestDataAttribute?: string
+  variants?: string[]
+}
+
```

Comment:
Just a note, but this `Button` file/component will be gone once we integrate Castor's button component with the Web SDK Customisation work.